examples/test-cra/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "cra-storybook",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "@kadira/storybook": "file:../../packages/react-storybook",
```

Comment:
This is to be able to do an actual npm install without linking via lerna.
So when everything is installed flat things should work as expected.